guide/ops/locations/_openstack.md

```diff
@@ -32,12 +80,16 @@ Do this by setting the desired values id as an option in the
 
 ### Floating IPs
 
-Configuration of floating IPs is as networks; specify the pools to use as another
-[template option](#custom-template-options):
+The `autoAssignFloatingIp` option means that a [floating ip](https://www.mirantis.com/blog/configuring-floating-ip-addresses-networking-openstack-public-private-clouds/)
```

Comment:
Could do - but the two mentions are in CloudStack and OpenStack pages. We could do with a general description of the options available (rather than just calling out a small number of specific ones for each cloud). When we write that, then we'll include this description. But not in this PR.